usr/local/www/diag_logs_settings.php

```diff
@@ -254,234 +254,298 @@ function check_everything() {
-					</tr>
-				</table>
-			</td>
-		</tr>
-		<tr>
-			<td width="22%" valign="top" class="vncell"><?=gettext("Remote Syslog Contents");?></td>
-			<td width="78%" class="vtable">
-				<input name="logall" id="logall" type="checkbox" value="yes" <?php if ($pconfig['logall']) echo "checked=\"checked\""; ?> onclick="check_everything();" />
-				<?=gettext("Everything");?><br /><br />
-				<input name="system" id="system" type="checkbox" value="yes" onclick="enable_change(false)" <?php if ($pconfig['system']) echo "checked=\"checked\""; ?> />
-				<?=gettext("System events");?><br />
-				<input name="filter" id="filter" type="checkbox" value="yes" <?php if ($pconfig['filter']) echo "checked=\"checked\""; ?> />
-				<?=gettext("Firewall events");?><br />
-				<input name="dhcp" id="dhcp" type="checkbox" value="yes" <?php if ($pconfig['dhcp']) echo "checked=\"checked\""; ?> />
-				<?=gettext("DHCP service events");?><br />
-				<input name="portalauth" id="portalauth" type="checkbox" value="yes" <?php if ($pconfig['portalauth']) echo "checked=\"checked\""; ?> />
-				<?=gettext("Portal Auth events");?><br />
-				<input name="vpn" id="vpn" type="checkbox" value="yes" <?php if ($pconfig['vpn']) echo "checked=\"checked\""; ?> />
-				<?=gettext("VPN (PPTP, IPsec, OpenVPN) events");?><br />
-				<input name="apinger" id="apinger" type="checkbox" value="yes" <?php if ($pconfig['apinger']) echo "checked=\"checked\""; ?> />
-				<?=gettext("Gateway Monitor events");?><br />
-				<input name="relayd" id="relayd" type="checkbox" value="yes" <?php if ($pconfig['relayd']) echo "checked=\"checked\""; ?> />
-				<?=gettext("Server Load Balancer events");?><br />
-				<input name="hostapd" id="hostapd" type="checkbox" value="yes" <?php if ($pconfig['hostapd']) echo "checked=\"checked\""; ?> />
-				<?=gettext("Wireless events");?><br />
-			</td>
-		</tr>
-		<tr>
-			<td width="22%" valign="top">&nbsp;</td>
-			<td width="78%"> <input name="Submit" type="submit" class="formbtn" value="<?=gettext("Save"); ?>" onclick="enable_change(true)" />
-			</td>
-		</tr>
-		<tr>
-			<td width="22%" height="53" valign="top">&nbsp;</td>
-			<td width="78%"><strong><span class="red"><?=gettext("Note:")?></span></strong><br />
-			<?=gettext("syslog sends UDP datagrams to port 514 on the specified " .
-			"remote syslog server, unless another port is specified. Be sure to set syslogd on the " .
-			"remote server to accept syslog messages from");?> <?=$g['product_name']?>.
-			</td>
-		</tr>
-	</table>
-	</div>
-</td></tr>
-</table>
-</form>
+
+<!-- Setting the button to class btn-danger makes it red, but also causes
+a confirmation dialog to be automatically presented when the button is clicked -->
+<script>
+	document.getElementById("resetlogs").className = "btn btn-danger btn-xs";
+</script>
+
 <script type="text/javascript">
 //<![CDATA[
 enable_change(false);
 //]]>
 </script>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+
+<?php include("foot.inc"); ?>
+
+<script>
```

Comment:
There should be no output after the *foot.inc*, setRemoteServers should be replaced by ->toggles but regardless, any scripts should be put before the foot. They probably won't work until you wrap them in ```events.push()```, but that is intentional